nvflare/private/fed/server/server_engine_internal_spec.py

```diff
@@ -206,15 +203,26 @@ def build_component(self, config_dict):
         """
         pass
 
+    @abstractmethod
+    def get_client_name_from_token(self, token: str) -> str:
+        """Get the registered client name from communication token.
+
+        Args:
+            token: communication token
+
+        Returns:
+            Client name
+        """
+        pass
+
```

Comment:
This function already defined in `ServerEngineSpec`

No need to define again.

Reply:
Removed.